pkg/alpine/Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN for repo in main community testing; do \
 
 # set the default repository to use
 RUN cp /mirror/${ALPINE_VERSION}/rootfs/etc/apk/repositories /etc/apk
+RUN cat /mirror/3.18/rootfs/etc/apk/repositories >> /etc/apk/repositories
```

Comment:
APLINE_VERSION is 3.16 so this means we'll have a mixture of 3.16 and 3.18? Wouldn't it be better to move to 3.18?

Reply:
We can move to 3.18 but replacing one package was just faster. Besides we are using go from edge branch which is not 3.16 either. But I agree, having consistent Alpine version across all our packages is better.

Reply:
OK, but we need to plan for an overall Alpine version update (in addition to the kernel update).
Raises the question whether we should move the kernel to 6.1 and skip 5.15 but that isn't the topic of this PR.

Reply:
I've created a ticket for that and it is already in progress